src/lib/Toast/Toast.svelte

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+  import { fly } from 'svelte/transition';
+  import { defaultToastProperties, type ToastProperties, type ToastType } from './properties';
+  import { createEventDispatcher, onDestroy, onMount } from 'svelte';
+
+  export let properties: ToastProperties = defaultToastProperties;
+
+  const dispatch = createEventDispatcher();
+
+  let showToast = false;
+  let leftIcon = properties.leftIcon ?? null;
+  let timeout: ReturnType<typeof setInterval>;
+
+  properties = {
```

Comment:
No need of doing this. 
If properties is exported, then passed params will be received in the same.

Reply:
updated.